packages/pyright-internal/src/common/pathUtils.ts

```diff
@@ -710,13 +710,13 @@ export function hasPythonExtension(path: string) {
     return path.endsWith('.py') || path.endsWith('.pyi');
 }
 
-export function getFileSpec(rootPath: string, fileSpec: string): FileSpec {
-    let regExPattern = getWildcardRegexPattern(rootPath, fileSpec);
+export function getFileSpec(fs: FileSystem, rootPath: string, fileSpec: string): FileSpec {
+    let regExPattern = normalizePathCase(fs, getWildcardRegexPattern(rootPath, fileSpec));
```

Comment:
This looks a little suspect. Does "normalizePathCase" handle regular expressions correctly?

Reply:
I looked what is added as regex pattern and none of them seems to be affected by lower casing, but I can take another approach where regex include /i flag rather than making whole thing lower case.

Reply:
OK, as long as you've considered all of the possible escaping behaviors and the impact that lower-casing may have on those cases.